src/tutorials/first_game_br.md

```diff
@@ -10,116 +10,162 @@ tags:
 
 1. In the **Project Content** window, search for the `Helper Plane` template.
-2. Drag the template into the **Hierarchy**.
-3. Right click the template and select **Deinstance this Template**
+2. Drag the template into the **Viewport** window.
+3. Right click the template and select **Deinstance this Template**.
+
+!!! tip "You can focus the camera on an object by selecting the object in the Hierarchy and pressing the ++F++ key."
 
-<!--- TODO add image of hierarchy --->
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/FrameworksBR/FrameworksBR_HelperPlaneAdded.mp4" type="video/mp4" />
+    </video>
+</div>
 
 ### Change the Colors
 
-Navigate to the **Helper_Plane > ClientContext > Geo** group in the **Hierarchy**. There are three components that make up the airplane. One at a time, select a component and change the **Color Override** property inside the **Properties** window.
+Inside the Hierarchy, navigate to the **Geo** group that is inside the ClientContext of the **Helper_Plane**. There are three components that make up the airplane. One at a time, select a component and change the **Color Override** property inside the **Properties** window.
 
-<!--- TODO add image of Color Override property --->
+![!Plane Color](../img/FrameworksBR/FrameworksBR_HelperPlaneColor.png){: .center loading="lazy" }
 
 ### Add a Decal
 
-You can also add more components to the airplane. Decals can be used to display a graphic on a surface.
+You can continue customizing the airplane by using decals. Decals can be used to display a graphic on a surface.
 
-1. Inside the **Core Content** window, find the **Art Objects > Decals** category.
-2. Drag and drop the `Decal Military Symbols 01` decal into the **Geo** group within the **Hierarchy**.
+1. Inside the **Core Content** window, find the **Art Objects** category and **Decals** subcategory.
+2. Drag and drop the **Decal Military Symbols 01** decal into the **Geo** group within the Hierarchy.
 3. Position the decal onto plane.
 4. Rotate and Scale the decal to your liking.
 
-<!--- TODO add image of decal on plane --->
+![!Plane Decal](../img/FrameworksBR/FrameworksBR_HelperPlaneDecal.png){: .center loading="lazy" }
 
 !!! info
-    Decals can also be modified such as changing the color. With the decal selected, open the **Properties** window and scroll down to the **Smart** category. Some decals have a **Shape Index** property that change the graphic to different variations.
+    Decals can also be modified such as changing the color. With the decal selected, open the **Properties** window and scroll down to the **Smart** category. Not all decals have the same properties. Some decals have a **Shape Index** property that change the graphic to different variations. For example, there are multiple military symbols available inside the **Decal Military Symbols 01** decal.
```

Comment:
"modified such as" isn't really grammatical. You could list the things you can modified. Since it's a tutorial, you could also start the sentence with "You can" to make it read more clearly. 

Really like the rest of these edits.